source/_patterns/00-atomer/01-forms/10-tekst-input.mustache

```diff
@@ -15,7 +15,7 @@
 
       <input class="form-control {{ form-control-class }} {{# form-label-hasIcon }} a-hasIcon {{/ form-label-hasIcon }}
         {{# form-input-hasButton }} a-hasButton {{/ form-input-hasButton }} {{# form-input-isPassword }} a-hasButton {{/ form-input-isPassword }}"
-        type="{{ input-type }}"
+        type="{{ input-type }}" {{ hasAutoFocus }}
```

Comment:
If this is to be named hasAutoFocus it should be a boolean. and rather do: {{# hasAutoFocus }} autofocus {{/ hasAutoFocus }}